projects/frontend/shared-components/gui/projects/shared/src/lib/common/interfaces/comparable.interface.ts

```diff
@@ -0,0 +1,72 @@
+
+
+/*
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * ** Interface for Comparison data.
+ *
+ * @author gorankokin
```

Comment:
Please, remove the internal username references from this and all the files below.

Reply:
Why?

Reply:
A username is not a secret? Also, we already have a bunch of author ids and emails.

Reply:
I'll just remove them, cause most likely nobody will contact the author directly for support.

Reply:
Iva, generally none of these stuff is a secret, but these brings no meaningful value outside of the context of VMW systems :). 

If I'm an external user, I guess I'll have hard times understanding who is 'gorankokin'. 

If we want to keep them, may be we should enhance the author information with name/e-mail potentially etc. stuff, which I believe is easier just to move out. 

But this is my personal opinion :), so I'll will resolve this thread and will leave you the decision how to proceed here. If you like to leave these as is, I do not mind.